source/refunding_payments/index.html.md.erb

```diff
@@ -176,21 +171,12 @@ Example response:
 
 `settled_date` will be either:
 
-- the date Stripe took the refund from the payments that Stripe sent to your bank account
-- missing if Stripe has not yet taken the refund
-
-All dates are in Coordinated Universal Time (UTC).
+* the date Stripe took the refund from the payments that Stripe sent to your bank account
+* missing if Stripe has not yet taken the refund
 
 ### Send email notifications about refunds
 
-You can use the GOV.UK Pay admin tool to turn on sending refund email notifications to your users. [Select the account in the GOV.UK admin tool](https://selfservice.payments.service.gov.uk/my-services), then select **Settings**.
-
-## Getting a list of refunds
-
-You can use the GOV.UK API to:
-
-- get all refunds for a single payment
-- search refunds
+You can use the GOV.UK Pay admin tool to turn on sending refund email notifications to your users. [Select your service in the GOV.UK admin tool](https://selfservice.payments.service.gov.uk/my-services), then select **Settings**.
```

Comment:
It might be worth adding extra detail to name the UI option users need to actually toggle to turn this on.